driver-sync/src/test/functional/com/mongodb/client/AbstractTransactionsTest.java

```diff
@@ -458,6 +460,14 @@ public Void execute() {
         }
     }
 
+    boolean processExtendedTestOperation(final BsonDocument operation, final ClientSession clientSession) {
```

Comment:
I don't understand the semantics of this method.  What does "extended" refer to? What's supposed to happen in overrides of this method, and what does the return value indicate?  It seems awkward, like the abstraction between base and subclasses is not well thought out.

Reply:
I have removed this method and consolidated the operations in `AbstractUnifiedTest`.